cssdb.json

```
@@ -373,6 +373,15 @@
       }
     ]
   },
+  {
+    "id": "in-out-of-range",
+    "title": "`:in-range` and `:out-of-range` Pseudo-Classes",
+    "description": "If a temporal or number <input> has max and/or min attributes, then :in-range matches when the value is within the specified range and :out-of-range matches when the value is outside the specified range.",
+    "specification": "https://drafts.csswg.org/selectors-4/#range-pseudos",
```

This has advanced as a Working Draft at w3.org.

```suggestion
    "specification": "https://www.w3.org/TR/selectors-4/#range-pseudos",
```